Source/Engine/UI/GUI/Common/TextBoxBase.cs

```diff
@@ -1296,6 +1299,42 @@ public override bool OnKeyDown(KeyboardKeys key)
             bool ctrDown = window.GetKey(KeyboardKeys.Control);
             KeyDown?.Invoke(key);
 
+            // Handle controls that have bindings
+#if FLAX_EDITOR
+            InputOptions options = FlaxEditor.Editor.Instance.Options.Options.Input;
+            if (options.Copy.Process(this))
```

Comment:
I'm not sure if we want to modify logic of gameplay component in Editor to have those shortcuts.

Reply:
> I'm not sure if we want to modify logic of gameplay component in Editor to have those shortcuts.

So just drop the deselect all keybind from the gameplay component? or am I misunderstanding?

Reply:
I mean that using Editor shortcuts in this `TextBoxBase` will work in editor but won't work in game so devs can treat it as an error.

Reply:
Oh, actually, it isn't there. I was misunderstanding what you are saying.
I modified the editor things just because I wanted to be able to customize keybinds for editor text boxes. Is there a way to get the scope of the control? As in if its part of the editor or part of the game inside the editor?

Reply:
We could leave this option for editor keybinds, just need t be aware that editor shortcuts won't work in cooked game. But ofc `TextBoxBase` implements typical `Ctrl+C`/`Ctrl+V`/etc. shortcuts as a base functionality.